mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -52,9 +32,31 @@
 import android.view.MenuItem;
 import android.view.View;
 import android.view.ViewGroup;
+
+import org.hawkular.client.android.R;
```

Comment:
Actually to avoid unused imports I optimised the imports.
That's how this reformation must have happened.
Will take care from now onwards